HARSHAL/javascript/assignment12/assignment12Q2.js

```diff
@@ -0,0 +1,55 @@
+class Employee {
+    constructor() {
+        this.name = "";
+        this.empno = 0;
+        this.address = "";
+        this.salary = 0;
+        this.da;
+        this.hra;
+        this.pf;
+        this.gross;
+        this.net;
```

Comment:
Same Here pass empty value.
```suggestion
        this.net = 0;
```